third_party/blink/renderer/core/fetch/global_fetch.cc

```diff
@@ -132,6 +134,15 @@ ScriptPromise GlobalFetch::fetch(ScriptState* script_state,
                                  const V8RequestInfo* input,
                                  const RequestInit* init,
                                  ExceptionState& exception_state) {
+  // Workaround for invalid window pointers being used when this function was called
+  // when no creation context is available for an unknown reason.
+  //
+  // See https://linear.app/replay/issue/TT-957
+  if (!LocalDOMWindowPointerIsValid(&window)) {
+    exception_state.ThrowTypeError("Invalid receiver.");
```

Comment:
Please add diagnostics here. I suggested how [here](https://linear.app/replay/issue/TT-957/recorder-crash-in-getcreationcontextchecked#comment-f2a8d32a).